docs/general/administration/hardware-acceleration/intel.md

```diff
@@ -345,7 +345,17 @@ Root permission is required.
 
 :::
 
-1. Assuming you have added the jellyfin repository to your apt source list and installed the `jellyfin-server` and `jellyfin-web`.
+1. Assuming you have added the jellyfin repository to your apt source list and installed the `jellyfin-server` and `jellyfin-web`, you will need to ensure Intel's media drivers are installed.
```

Comment:
In fact, our jellyfin-ffmpeg deb package already contains the latest intel-media-driver, which is iHD. `/usr/lib/jellyfin-ffmpeg/lib/dri/iHD*.so`.

The iHD driver from LTS distro can easily become outdated and miss support for new hardware.